packages/cfd/src/Containers/cfd-password-modal/cfd-password-form/create-password.tsx

```diff
@@ -0,0 +1,120 @@
+                        </Text>
+                        <Text size='xs' align='center' className='cfd-password-modal__create-password-description'>
+                            <Localize
+                                i18n_default_text='You can use this password for all your {{platform}} accounts.'
+                                values={{
+                                    platform: getCFDPlatformLabel(platform),
+                                }}
+                            />
+                        </Text>
+                        <div className='input-element'>
+                            <PasswordMeter
+                                input={values.password}
+                                has_error={!!(touched.password && errors.password)}
+                                custom_feedback_messages={getErrorMessages().password_warnings}
+                            >
+                                <PasswordInput
+                                    autoComplete='new-password'
+                                    label={localize('{{platform}} password', {
+                                        platform: getCFDPlatformLabel(platform),
+                                    })}
+                                    error={
+                                        (touched.password && errors.password) ||
+                                        (values.password.length === 0 ? error_message : '')
+                                    }
+                                    name='password'
+                                    value={values.password}
+                                    onBlur={handleBlur}
+                                    onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                        handlePasswordInputChange(e, handleChange, validateForm, setFieldTouched);
+                                    }}
+                                    data_testId={`dt_${platform}_password`}
+                                />
+                            </PasswordMeter>
+                        </div>
+                        {is_real_financial_stp && (
+                            <div className='dc-modal__container_cfd-password-modal__description'>
+                                <Localize i18n_default_text='Your MT5 Financial STP account will be opened through Deriv (FX) Ltd. All trading in this account is subject to the regulations and guidelines of the Labuan Financial Service Authority (LFSA). None of your other accounts, including your Deriv account, is subject to the regulations and guidelines of the Labuan Financial Service Authority (LFSA).' />
+                            </div>
+                        )}
+                        <FormSubmitButton
+                            is_disabled={!values.password || Object.keys(errors).length > 0}
+                            is_loading={isSubmitting}
+                            label={localize('Create {{platform}} password', {
```

Comment:
Just an opinion, to be handled in a different card but can't we modify the FormSubmitButton label prop to take string as well as JSX.Element. If JSX.Element it should be rendered as Button's children ?
This way we can pass Localize component to the FormSubmitButton.
@mahdiyeh-deriv